migrations/20220105195635_init.ts

```diff
@@ -0,0 +1,36 @@
+import { Knex } from 'knex';
+
+// Funny how this number ended up being the last changelog migration...
+const MAGIC_CHANGELOG_NUMBER = 42;
```

Comment:
Well.. It is the `Answer to the Ultimate Question of Life, the Universe, and Everything,` :rofl: